rasa/train.py

```diff
@@ -399,15 +477,36 @@ async def _train_core_with_validated_data(
         domain, config = await asyncio.gather(
             file_importer.get_domain(), file_importer.get_config()
         )
+
         async with telemetry.track_model_training(file_importer, model_type="core"):
-            await rasa.core.train(
-                domain_file=domain,
-                training_resource=file_importer,
-                output_path=os.path.join(_train_path, DEFAULT_CORE_SUBDIRECTORY_NAME),
-                policy_config=config,
-                additional_arguments=additional_arguments,
-                interpreter=interpreter,
-            )
+            if training_in_chunks:
+                await rasa.core.train_in_chunks(
```

Comment:
optional: Do you think it makes sense to have something like an interface for core training so that we could do something like this:
```python
if training_in_chunks:
  trainer = RasaCoreChunkTrainer(number_of_chunks)
else:
  trainer = RasaCoreTrainer()

trainer.train(...)
```

Reply:
Not sure. I guess that is something I would consider when we actually implement the core training. So something for another PR.